pulsar-broker/src/main/java/com/yahoo/pulsar/broker/admin/Brokers.java

```diff
@@ -79,4 +95,79 @@
             throw new RestException(e);
         }
     }
+    
+    @POST
+    @Path("/configuration/{configName}/{configValue}")
+    @ApiOperation(value = "Update broker service configuration. This operation requires Pulsar super-user privileges.")
+    @ApiResponses(value = { @ApiResponse(code = 204, message = "Service configuration updated successfully"),
+            @ApiResponse(code = 403, message = "You don't have admin permission to update service-configuration"),
+            @ApiResponse(code = 404, message = "Configuration not found"),
+            @ApiResponse(code = 412, message = "Configuration can't be updated dynamically") })
+    public void updateConfiguration(@PathParam("configName") String configName, @PathParam("configValue") String configValue) throws Exception{
+        validateSuperUserAccess();
+        updateServiecConfiguration(configName, configValue);
+    }
+
+    /**
+     * if {@link ServiceConfiguration}-field is allowed to be modified dynamically, update configuration-map into zk, so
+     * all other brokers can see the change and take appropriate action on the change.
+     * 
+     * @param configName
+     *            : configuration key
+     * @param configValue
+     *            : configuration value
+     */
+    private void updateServiecConfiguration(String configName, String configValue) {
+        try {
+            Field field = ServiceConfiguration.class.getDeclaredField(configName);
```

Comment:
Instead of doing the reflection check on each request, could we doing just once (at startup) and keep a `Set<>` of keys that are dynamically updatable?